src/components/Clients/Brandings.tsx

```diff
@@ -0,0 +1,21 @@
+import Google from './../../assets/images/brandings/google.svg'
+import Github from './../../assets/images/brandings/github.svg'
+import Windows from './../../assets/images/brandings/windows.svg'
+import Amazon from './../../assets/images/brandings/amazon.svg'
+import Meta from './../../assets/images/brandings/meta.svg'
+import { BrandingsWrapper } from './Clients.styles'
+import React from 'react'
+
+const Brandings: React.FC = () => {
+  return (
+    <BrandingsWrapper>
+      <img className="branding-img" src={Meta} />
```

Comment:
add this brands in an array and render from them, because when the backend will be ready, the brands will come in array